src/main/java/cubicchunks/worldgen/generator/custom/populator/PopulatorUtils.java

```diff
@@ -75,17 +75,23 @@ public static void genOreUniform(ICubicWorld world, CustomGeneratorSettings cfg,
         }
     }
 
-    public static void genOreGaussian(ICubicWorld world, CustomGeneratorSettings cfg, Random random, CubePos pos,
-            int count, double probability, WorldGenerator generator, double mean, double stdDev) {
+    public static void genOreBellCurve(ICubicWorld world, CustomGeneratorSettings cfg, Random random, CubePos pos, int count,
+                                      double probability, WorldGenerator generator, double mean, double stdDev, double spacing, double minY, double maxY) {
+
+        int minBlockY = Math.round((float) (minY * cfg.heightFactor + cfg.heightOffset));
+        int maxBlockY = Math.round((float) (maxY * cfg.heightFactor + cfg.heightOffset));
+        int iSpacing = Math.round((float) (spacing * cfg.heightFactor + cfg.heightOffset));
+        int iMean = Math.round((float) (mean * cfg.heightFactor + cfg.heightOffset));
         for (int i = 0; i < count; ++i) {
-            if (random.nextDouble() > probability) {
-                continue;
-            }
             int yOffset = random.nextInt(Cube.SIZE) + Cube.SIZE / 2;
             int blockY = pos.getMinBlockY() + yOffset;
-            //noinspection SuspiciousNameCombination
-            double prob = MathUtil.gaussianProbabilityDensity(blockY, mean, stdDev);
-            if (random.nextDouble() > prob) {
+            //skip all potential spawns outside the spawn range
+            if((blockY > maxBlockY) || (blockY < minBlockY)){
+                continue;
+            }
+            double modifier = MathUtil.bellCurveProbabilityCyclic(blockY, iMean, stdDev, iSpacing);
```

Comment:
I still don't see the scaling applied to stdDev.

Reply:
stdDev doesn't work on a block scale. If I used that scaling I would only create a bar from -0.99 to -0.75 or so, else the belts become extremely wide. That's why it's just a direct float value.

Reply:
How is that not in blocks? https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule

Currently changing world generation height will keep the standard deviation the same amount of blocks. Just divide the value by 64 and multiply by heightFactor (*actually, I just realized that I added methods to get the appropriate values, use those. They are more accurate and in the future will be user adjustable).

Reply:
Ok, in a way it is in blocks. It just doesn't directly visibly translate into blocks like the other settings do.
If I get it correctly, you added an option to scale everything that is being generated vertically, which you want to put onto the belt width too. Then I still don't apply your hight scaling system, only the scaling factor (which gets normalized by vanilla value).
Still I don't think it's sensible to have a slider going from 0.99 to 0.75, but if you want it that way, I can change that too.

Reply:
Make it shot % of world height, between 0 and 100%.

Reply:
I will need to give the spacing slider a longer range then. So you want to calculate it as
stdDev = sliderValue * 64 * heightFactor ?
(Assuming height factor is a factor to scale the world height based on the vanilla 64 scale)

Reply:
No, `stdDev` should just be `sliderValueAs0To1 * cfg.getMaxHeightOffset()`.
And you should divide the current value of your constant by 64 because that's the default value of maxHeightOffset )which reduces to value of height factor for default settings).

Reply:
Changed it.